CHANGELOG.asciidoc

```diff
@@ -213,6 +213,10 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha1...v7.0.0-alpha2[Check the
 - Always include Pod UID as part of Pod metadata. {pull}9517[9517]
 - Autodiscovery no longer requires that the `condition` field be set. If left unset all configs will be matched. {pull}9029[9029]
 - Add geo fields to `add_host_metadata` processor. {pull}9392[9392]
+- Always include Pod UID as part of Pod metadata. {pull]9517[9517]
+- Optional autodiscovery hints separator config for docker object labels. {issue}9545[9545] {pull}9547[9547]
```

Comment:
We have added a `CHANGELOG.next.asciidoc` file to add new changes, could you move the changelog line to this file?